src/main/java/io/vertx/core/net/PemKeyCertOptions.java

```diff
@@ -116,82 +139,234 @@ public JsonObject toJson() {
   }
 
   /**
-   * Get the path to the key file
+   * Get the path to the first key file
    *
    * @return the path to the key file
    */
+  @GenIgnore
   public String getKeyPath() {
```

Comment:
it's not deprecated, the @GenIgnore is rather here to support properly multiple paths.